### DISCUSSION TITLE
Customize the "No data"

### DISCUSSION THREAD

**Discussion by GreenRover** [grafana/grafana#51474] [Dashboards, Panels & Visualizations]
If you have a dashboard where you like to report problems.
The good state is  to have nothing to display.

There for it would help to be able to customize the the "No data" message per panel.
In my use case i would like to display "Currently no problems" instead of "No data" if my promql query not return any metrics.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
